projectile.el

```diff
@@ -3128,6 +3124,11 @@ a manual COMMAND-TYPE command is created with
                                   :test #'projectile--cmake-test-command
                                   :install #'projectile--cmake-install-command
                                   :package "cmake --build build --target package")
+;; Go
```

Comment:
Might be good to add a comment this should have higher priority than Make.

Reply:
Thanks for pointing that out.  Done!